modules/plugin/shapefile/src/main/java/org/geotools/data/shapefile/ShapefileDumper.java

```diff
@@ -333,6 +351,16 @@ private void writeToShapefile(
      */
     protected void shapefileDumped(String fileName, SimpleFeatureType remappedSchema)
             throws IOException {
+        shapefileDumped(remappedSchema);
```

Comment:
I'd add in the javadocs that this method default implementation just calls the other, should helps sub-classers decide which method to override.